README.md

```diff
@@ -137,24 +137,41 @@ Prefix with selected service, for example "!ig add accountname" or "!twitter lis
 
 * add [accountname] - Add account to this room (Must be done as room admin)
 * del [accountname] - Delete account from room (Must be done as room admin)
-* list - List accounts in room 
+* list - List accounts in room
 * poll - Poll for new items  (Must be done as bot owner)
 * clear - Clear all accounts from this room  (Must be done as room admin)
 
 #### Instagram
 
 Polls instagram account(s). Uses instagram scraper library
-without any authentication or api key. 
+without any authentication or api key.
 
 See: https://github.com/realsirjoe/instagram-scraper/
 
 #### Twitter
 
 Polls twitter account(s). Uses twitter scraper library
-without any authentication or api key. 
+without any authentication or api key.
 
 See: https://github.com/taspinar/twitterscraper/tree/master/twitterscraper
 
+#### Url
+
+Watches all messages in a room and if a url is found tries to fetch it and
+spit out the title if found.
+
```

Comment:
Is this on by default or does it need to be disabled? Please tell here.

Reply:
Added note that this is disabled by default and needs to be activated for every room this is wanted.